### DISCUSSION TITLE
Проблема с гирляндой: светится только 2 первых светодиода

### DISCUSSION THREAD

**Discussion by gistol** [vvip-68/LedPanelWiFi#48] [Вопросы и ответы]
Приветствую. Можете подсказать в чём может быть проблема. Всё подключил, собрал, завёл веб сервер, но при подключении гирлянды в пин D4 на плате NodeMcu Lua CH340 https://aliexpress.ru/item/32659028315.html?spm=a2g2w.orderdetail.0.0.51244aa6aBjPzc&sku_id=12000028578582196 горят только 2 первых светодиода(1-й белый, 2-й зеленый), при перезагрузке устройства, бывает загорается один-первый светодиод белым. Версия ядра ESP 3.1.2, FastLEd пробовал 3.5 и 3.6. Гирлянда подключена в D4 через резистор 200 ОМ. Пробовал на 2х разных гирляндах, проблема одна и та же.

**Comment by vvip-68** [comment]
А в настройках линии вывода - куда сигнал назначен? По умолчанию там вывод на D2 стоит.

**Comment by gistol** [comment]
На D2 и стоит, но если подключить на пин 2, то вообще ничего не происходит.

**Comment by gistol** [comment]
В прошивке 
```
#define WIDTH 5               
#define HEIGHT 64              
#define MATRIX_TYPE 0           
#define MATRIX_INDEX 1          
#define CONNECTION_ANGLE 0      
#define STRIP_DIRECTION 0       
#define META_MATRIX_WIDTH 1    
#define META_MATRIX_HEIGHT 1   
#define META_MATRIX_TYPE 0     
#define META_MATRIX_ANGLE 0    
#define META_MATRIX_DIRECTION 0
#define COLOR_ORDER       GRB   
#define LED_CHIP      WS2812B   
#define DEVICE_TYPE 1          
#define BUTTON_TYPE 1           
#define USE_BUTTON 0            
#define USE_POWER 0             
#define POWER_ON  HIGH         
#define POWER_OFF LOW           
#define USE_ALARM 0             
#define ALARM_ON  HIGH          
#define ALARM_OFF LOW           
#define USE_AUX 0              
#define AUX_ON  HIGH            
#define AUX_OFF LOW             
#define USE_TM1637 0           
#define USE_WEATHER 0           
#define USE_E131 0             
#define BIG_FONT 0            
#define USE_SD 0               
#define FS_AS_SD 0                                 
#define WAIT_PLAY_FINISHED 1  
#define REPEAT_PLAY 0          
#define USE_MP3 0               
#define DFPLAYER_TYPE 0         
#define GUARD_DELAY 75          
#define LED_PIN D2             
#define PIN_BTN D4                                          
#define SD_CS_PIN D8            
#define STX D3                
#define SRX D4                 
#define DIO D5                
#define CLK D7                 
#define POWER_PIN D1            
#define ALARM_PIN D0           
#define AUX_PIN D0            
#define DEBUG_SERIAL 1 
```

**Reply by vvip-68** [reply]
Тут правильнее смотреть в Web-интерфейсе что реально используется и какие пины там назначены

**Comment by gistol** [comment]
В общем,через n-ое количество попыток завёл ленту через ПИН D2. Гирлянда зажглась, только при сбросе питания ленты.

**Comment by gistol** [comment]
У вас есть донатная строка, где можно поддержать проект?

**Reply by vvip-68** [reply]
На странице проекта по ссылке Sponsor - вторая ссылка YandexDisk - картинка с номером карточки сбербанка

**Reply by vvip-68** [reply]
@gistol, спасибо за поддержку проекта!
